spec.html

```diff
@@ -3008,6 +3008,8 @@ <h1>Await</h1>
           1. Perform ! PerformPromiseThen(_promiseCapability_.[[Promise]], _onFulfilled_, _onRejected_).
           1. Remove _asyncContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
           1. Set the code evaluation state of _asyncContext_ such that when evaluation is resumed with a Completion _completion_, the following steps of the algorithm that invoked Await will be performed, with _completion_ available.
+          1. Return.
+          1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _asyncContext_.
```

Comment:
this wording seems really confusing to me; so much so that i don't have a better suggestion because i don't know what it's trying to say

Reply:
This wording is already extant in other places (two other places in the spec in fact), and recommended to me for use by @zenparsing earlier in the thread. The whole topic of execution context switching is super complicated, so this is pretty much as good as it gets.

Reply:
Yes, well said. See GeneratorYield for another usage of this kind of note.